zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -32,12 +33,20 @@ abstract class HttpRunnableSpec extends DefaultRunnableSpec { self =>
     method: Method = Method.GET,
     content: String = "",
     headers: Headers = Headers.empty,
+    httpVersion: HttpVersion = HttpVersion.HTTP_1_1,
```

Comment:
this should be the first param

Reply:
this will require changing other overloaded methods across tests.

Reply:
@tusharmath  Please suggest,Ideally version should be the first parameter.